jstz_proto/src/executor/contract.rs

```diff
@@ -133,8 +133,21 @@ impl Script {
         Ok(Self(module))
     }
 
-    fn register_apis(&self, contract_address: Address, context: &mut Context<'_>) {
+    fn register_apis(
+        &self,
+        contract_address: Address,
+        context: &mut Context<'_>,
+        operation_hash: &OperationHash,
+    ) {
         register_web_apis(self.realm(), context);
+        // TODO: Register console API in `register_web_apis` once `Jstz` object is implemented
+        self.realm().register_api(
+            jstz_api::ConsoleApi::Proto {
+                contract_address: contract_address.clone(),
+                operation_hash: operation_hash.clone(),
+            },
+            context,
+        );
```

Comment:
Update: This doesn't cause the console API to be registered twice. But it does permit the user to revert logging back to the Proto (JSON) style of logging instead of the Cli (Pretty) style of logging. This will be fixed in a future PR.